manifests/server.pp

```diff
@@ -679,6 +679,9 @@
   }
 
   if $ldap_enabled == true {
+    package {$openvpn::params::ldap_auth_plugin_package:
```

Comment:
can you add ldap_auth_plugin_package as additional param to the server class?

Reply:
Hi, yeah sure. Thanks!